### ISSUE TITLE
Overlay FPS issue.

### ISSUE THREAD

**Issue by Marshall-67** [IMXNOOBX/cs2-external-esp#4]
Hi there. ive been working with your code for a week now messing around.

the major fps issue comes from LWA_COLORKEY

    SetLayeredWindowAttributes(hwnd, RGB(0, 0, 0), 255, LWA_ALPHA);
    MARGINS margin = { -1 };
    DwmExtendFrameIntoClientArea(hwnd, &margin);
    
    
    This will fix it but i think you need to add some stuff however it makes the ESP look really transparent in light which i haven't found away around yet.
    
    
    May i ask why you dropped the discord version? seems to work alot better and i think you can bypass the block by running as admin and the launch option for discord overlay doesnt even drop trust factor. that was a myth

**Comment by IMXNOOBX** [comment]
Looks promising. If you have a demo to test what you said about the LWA_COLORKEY let me know so i can look into it

**Comment by VilotStar** [comment]
Regarding the Discord version, I haven't been able to get discord to inject into cs2 without -insecure being used, that is most likely the reason the discord branch was dropped, however you could use Nvidia overlay or a window overlay to fix lag I imagine.

**Comment by IMXNOOBX** [comment]
> Regarding the Discord version, I haven't been able to get discord to inject into cs2 without -insecure being used, that is most likely the reason the discord branch was dropped, however you could use Nvidia overlay or a window overlay to fix lag I imagine.

You are right. discord overlay is blocked by valve since csgo (that i know). I will try doing a new version with directx overlay. I don't have nvidia overlay and many people won't have it either so it will be easier to just do it in dx

**Comment by VilotStar** [comment]
That would make for a pretty versatile esp, I'm working on one right now that utilises handle hijacking and possibly steam overlay to display the esp, that should be pretty undetectable

**Comment by Qanbaz** [comment]
Any updates on this ?
